transform/pipelines/README.md

```diff
@@ -0,0 +1,2 @@
+TODO: explain that this is where the repos are going to get checked out:
```

Comment:
worth doing this TODO?

Reply:
It's gonna be done in that ticket below, I'll make this text more clear